src/libraries/Native/Unix/System.Native/pal_signal.h

```diff
@@ -40,8 +40,8 @@ typedef enum
     PosixSignalSIGQUIT = -3,
     PosixSignalSIGTERM = -4,
     PosixSignalSIGCHLD = -5,
-    PosixSignalSIGWINCH = -6,
-    PosixSignalSIGCONT = -7,
+    PosixSignalSIGCONT = -6,
+    PosixSignalSIGWINCH = -7,
```

Comment:
For reference, these values map to https://github.com/dotnet/runtime/blob/721a802c12b524912b6c398313d8eb5e73dec2ef/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/PosixSignal.cs#L28-L33.